cves/CVE-2020-7471.yml

```diff
@@ -103,10 +103,10 @@ unit_tested:
 
     For the fix_answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: This would be hard to catch in unit tests, since if you could predict the ways your sanitizer could break, you'd just make a better sanitizer.
+  fix: false
```

Comment:
I see that one unit test was added in the fix commits.

Reply:
I missed that, I'll review and see how it helped